proto/rpc/v1/submit.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package rpc.v1;
+
+// A request to submit the signed transaction to the ledger.
+message SubmitTransactionRequest {
+    // The signed transaction to submit.
+    bytes signed_transaction = 1;
+
+    bool fail_hard = 2;
+}
+
+// A response when a signed transaction is submitted to the ledger.
+message SubmitTransactionResponse {
```

Comment:
`txBlob` and `status` are missing from this message

Reply:
I don't see `status` as part of the response to this api [https://xrpl.org/submit.html](https://xrpl.org/submit.html)
`txBlob` is purposefully omitted; see my other comment.

Reply:
@keefertaylor that `status` field is part of every RPC and isn't specific to any particular API. I think it just means whether the RPC succeeded. gRPC has it's own status codes, which I am using.